file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -0,0 +1,300 @@
+      "! <p class="shorttext">Operation at Activation</p>
+      "! Data records which are to be deleted at activation are flagged with value 'L'
+      "! $values {@link zif_aff_scp1_v1.data:co_operation_at_activation}
+      operation_at_activation TYPE ty_operation_at_activation,
+      "! <p class="shorttext">Data</p>
+      "! Complete data record of selected key value
+      "! $required
+      data                    TYPE ty_row_data,
+      "! <p class="shorttext">Translations</p>
+      "! Translated data records of Table or View included in a BC Set
+      translations            TYPE ty_translations,
+    END OF ty_record.
+
+  "! <p class="shorttext">Data Records</p>
+  TYPES ty_records TYPE SORTED TABLE OF ty_record WITH UNIQUE KEY key_value.
+
+  TYPES:
+    "! <p class="shorttext">Attributes of Table/View Field</p>
+    "! You can define the behavior for field of table/view during activation
+    BEGIN OF ty_field_attribute,
+      "! <p class="shorttext">Field Name</p>
+      "! Name of field in Table/View
+      "! $required
+      field_name      TYPE c LENGTH 30,
+      "! <p class="shorttext">Field Attribute</p>
+      "! Defines the attribute of field like during BC Set activation, the field value must be copied to database
+      "! table or not
+      "! $values {@link zif_aff_scp1_v1.data:co_field_attribute}
+      "! $required
+      additional_info TYPE ty_additional_info,
+    END OF ty_field_attribute.
+
+  TYPES ty_field_attributes TYPE SORTED TABLE OF ty_field_attribute WITH UNIQUE KEY field_name.
+
+  TYPES:
+    "! <p class="shorttext">Entity</p>
+    "! Details of selected View or Table
+    BEGIN OF ty_entity,
+      "! <p class="shorttext">Entity Name</p>
+      "! Table name of each data record
+      "! $required
+      table_name       TYPE c LENGTH 30,
+      "! <p class="shorttext">Field Attributes</p>
+      "! You can define the behavior for field of table/view during activation
+      "! $required
+      field_attributes TYPE ty_field_attributes,
+      "! <p class="shorttext">Data Records</p>
+      "! Data records of Table or View included in a BC Set
+      "! $required
+      data_records     TYPE ty_records,
+    END OF ty_entity.
+
+  "! <p class="shorttext">Tables and Views for Selected Customizing Object</p>
+  TYPES ty_entities TYPE SORTED TABLE OF ty_entity WITH UNIQUE KEY table_name.
+
+  TYPES:
+    "! <p class="shorttext">Customizing Object</p>
+    "! Details of selected customizing object
+    BEGIN OF ty_customizing_object,
+      "! <p class="shorttext">Name</p>
+      "! Collections of tables which constitute logical business entities,
+      "! and must therefore be maintained and exported together.
+      "! $required
+      object_name  TYPE c LENGTH 30,
+      "! <p class="shorttext">Type</p>
+      "! Object type is used to classify customizing objects
+      "! $values {@link zif_aff_scp1_v1.data:co_object_type}
+      "! $required
+      object_type  TYPE ty_object_type,
```

Comment:
I am not sure whether we shall change the order to `object_type` `object_name`.

Reply:
I kept the same order in which the SCPR3 transaction displays these 2 fields. Also in the same sequence these 2 fields are displayed in transaction SOBJ + the table OBJH.